templates/terms_and_conditions_snippet.html

```diff
@@ -0,0 +1,76 @@
+<h4>INFORMED CONSENT</h4>
```

Comment:
We made this so we could use it in our terms and conditions page. Please include this file there too.